boa3_test/test_drive/model/invoker/invokeresult.py

```diff
@@ -0,0 +1,12 @@
+class _NotExecuted:
+    def __repr__(self) -> str:
+        return 'Invoke not executed yet'
+
+
+class _Canceled:
+    def __repr__(self) -> str:
+        return 'Invoke was canceled before being executed'
+
+
+NOT_EXECUTED = _NotExecuted()
+CANCELED = _Canceled()
```

Comment:
A way to identify that the invoke result was set using the logic in the test runner. Implemented classes instead of using literal string or StrEnum to support modifications if needed (we can revert it if there is no need for modification when the refactoring is done)
also the str on those classes is only used to better visualize during debug, since only `__repr__` is set (not `__str__`), so this string data is not really used